AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -99,7 +100,7 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
         }
     }
     
-    func deleteStreams(_ sessionUUID: SessionUUID) throws {
+    func updateSession(_ sessionUUID: SessionUUID) throws {
```

Comment:
Because it's used for both deleting streams/deleting sessions and updating edited sessions. It's the same endpoint and I didn't want to duplicate this method to just have it with different name :)

Reply:
Where is it used to update edited session? I don't see it, and looking at this method I only see that it deletes streams and then forces session update, which is only needed here, because the changes are introduced to streams and not session directly

Reply:
ugh, goood catch. No, I didn't want to change that one. It must have changed the name when I was refactoring this from deleteSession using "refactor". Thanks, changing it back to `deleteStreams`
`    func updateSession(session: SessionEntity, completion: @escaping () -> Void) {
`